### PR TITLE
Provide message id in the return

### PR DESCRIPTION
**Pull request by jliphard**
Add message ID to function return

### CODE REVIEW

File: exponent_server_sdk/__init__.py [hunk 1]
```diff
@@ -292,7 +292,8 @@ def _publish_internal(self, push_messages):
                 # If there is no status, assume error.
                 status=receipt.get('status', PushResponse.ERROR_STATUS),
                 message=receipt.get('message', ''),
-                details=receipt.get('details', None)))
+                details=receipt.get('details', None),
+                id=receipt.get('id', '')))
```
Comment (hanc1208) [inline exponent_server_sdk/__init__.py:296]: I think that `id` should be `None` when `id` does not exist

Comment (maguayo) [inline exponent_server_sdk/__init__.py:296]:
```suggestion
                id=receipt.get('id', None)))
```

**Comment by julianpacker** [comment]
When will this change get merged into master?

File: exponent_server_sdk/__init__.py [hunk 2]
```diff
@@ -292,7 +292,8 @@ def _publish_internal(self, push_messages):
                 # If there is no status, assume error.
                 status=receipt.get('status', PushResponse.ERROR_STATUS),
                 message=receipt.get('message', ''),
-                details=receipt.get('details', None)))
```
Comment (faridlopez) [inline exponent_server_sdk/__init__.py:295]: Replacing 'details' for 'id' also works. I don't think details is returning anything.

**Comment by faridlopez** [comment]
Is there an ETA for this code to be merge? or should we use our own modification?
I think is good for the community to maintain this code up to date.
Thanks.

**Comment by julianpacker** [comment]
I agree with you @faridlopez . Actually I made several changes in one the latest pull requests because I needed to make changes. Feel free to "pull" off that. I can try to get in touch with the developer and see if I can overtake ownership of this package

**Comment by julianpacker** [comment]
I've made this change in my pull request: https://github.com/expo-community/expo-server-sdk-python/pull/31/files. Please review if you have any concerns.

I think it's safe to close this pull request. If someone disagrees, reopen the pull request or comment here.

@faridlopez I'll be releasing a new package soon, so if you want to wait for the PyPi package, or just pull from master when I merge these pull requests in.

**Pull request closed without merging**